src/components/Shopping/ShoppingCart.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import data from "../../../data/products.json";
+import styled from "styled-components";
+import Image from "next/dist/client/image";
+
+export const ShoppingCart = (props) => {
+  return props.shoppingCart.map(({ id, variant }) => {
+    const product = data.find((product) => id === product.id);
+    return (
+      <StyledCart key={id}>
+        <h4>{product.name}</h4>
+        <p>{product.description}</p>
+        <p>
+          {product.variants[variant].price} € - {" "}
+          {product.variants[variant].duration} Tage
+        </p>
+      </StyledCart>
+    );
+  });
+};
```

Comment:
You can make use of the NumberFormatter I showed above.
You can initialize it outside of the component (`new Intl.Num…`) and then use the `format` function in your component.

NumberFormat has a very consistent way to display currencies and other numbers without you needing to worry about formatting.

```suggestion
const euroFormatter = new Intl.NumberFormat('de-de', { style: 'currency', currency: 'EUR' });

export const ShoppingCart = (props) => {
  return props.shoppingCart.map(({ id, variant }) => {
    const product = data.find((product) => id === product.id);
    return (
      <StyledCart key={id}>
        <h4>{product.name}</h4>
        <p>{product.description}</p>
        <p>
          {euroFormatter.format(product.variants[variant].price)} - {" "}
          {product.variants[variant].duration} Tage
        </p>
      </StyledCart>
    );
  });
};
```